releasenotes/notes/handle-deferred-close-forwarder.yaml

```
@@ -0,0 +1,4 @@
+---
+security:
+  - |
+    Handle the return value of Close() for writable files in `pkg/forwarder`
```

Changelog entries are RST, not markdown, and use double backticks for monospace text:

```suggestion
    Handle the return value of Close() for writable files in ``pkg/forwarder``
```